508. Most Frequent Subtree Sum.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val) {
+ *     this.val = val;
+ *     this.left = this.right = null;
+ * }
+ */
+/**
+ * @param {TreeNode} root
+ * @return {number[]}
+ */
+let findFrequentTreeSum = function(root) {
+    let myMap = new Map()
+    let max = 0
+    let output = []
+    
+    let loop = function(root) {
```

Comment:
I don't really like the word `loop` for this because it implies iteration, when this is recursive. Can we change it to `walk` or `traverse` or something along those lines?